### DISCUSSION TITLE
PQC Related: TLS support for Hybrid algorithms (X25519Kyber768Draft00 and X25519MLKEM768)

### DISCUSSION THREAD

**Discussion by Sanket2211** [bcgit/bc-java#1879] [Q&A]
I see that there is support for pure MLKEM TLS in the 1.79 version but when is the support for the hybrid algorithms planned (X25519Kyber768Draft00 and X25519MLKEM768) to released. Will it be included in the upcoming 1.79 release?

**Comment by prathamlashkari** [comment]
may be they will add in future
